bin/inx-chronicle/src/api/stardust/indexer/responses.rs

```diff
@@ -9,7 +9,7 @@ use crate::api::responses::impl_success_response;
 #[serde(rename_all = "camelCase")]
 pub struct IndexerOutputResponse {
     pub ledger_index: u32,
-    pub output_id: String,
+    pub items: Vec<String>,
 }
```

Comment:
We should be able to just reuse the `IndexerOutputsResponse` from below. The TIP actually uses the same schema for both:

<img width="956" alt="image" src="https://user-images.githubusercontent.com/3404250/186166774-0cf77cb7-730c-4644-80f5-c4d141471778.png">